### PR TITLE
[ADD] stock pickup date

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I will call it _stock_picking_pickup_date_ to specify better its function.

File: stock_picking_pickup_date/stock_move_view.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <filter name="pickup_14days" string="14 days"
+              domain="['&amp;', ('pickup_date', '&gt;=', context_today().strftime('%%Y-%%m-%%d')),
+                                ('pickup_date', '&lt;', (context_today() + relativedelta(days=14)).strftime('%%Y-%%m-%%d'))]" />
+          </group>
+        </xpath>
+      </field>
+    </record>
+
+    <!-- Override move tree with our search view -->
+    <record id="stock.action_out_picking_move" model="ir.actions.act_window">
```
Comment (pedrobaeza) [inline stock_picking_pickup_date/stock_move_view.xml:46]: You don't need to overwrite entire view, only the affected fields.

**Comment by pedrobaeza** [comment]
Please remove company logo.

**Comment by max3903** [comment]
Can you also add the pickup date field in the stock move form view (before the scheduled date) ?

Thank you.

**Comment by laetitia-gangloff** [comment]
Can you rebase, please ?

**Comment by pedrobaeza** [comment]
Closing as it has no answer

**Pull request closed without merging**